src/App.tsx

```diff
@@ -77,7 +86,7 @@ export const App: React.FC = () => {
                   <PostsList
```

Comment:
Verify that the `PostsList` component is correctly implemented and receives the expected props.